src/views/Timesheets/components/ShiftItem/index.js

```diff
@@ -10,15 +9,15 @@ import {
   TextField,
   Tooltip,
   Typography,
-} from '@material-ui/core';
-import { withStyles } from '@material-ui/core/styles';
-import CheckOutlinedIcon from '@material-ui/icons/CheckOutlined';
-import ClearOutlinedIcon from '@material-ui/icons/ClearOutlined';
-import DeleteForeverOutlinedIcon from '@material-ui/icons/DeleteForeverOutlined';
-import EditOutlinedIcon from '@material-ui/icons/EditOutlined';
-import InfoOutlinedIcon from '@material-ui/icons/InfoOutlined';
-import MessageOutlinedIcon from '@material-ui/icons/MessageOutlined';
-import { DateTimePicker, MuiPickersUtilsProvider } from '@material-ui/pickers';
+} from '@mui/material';
+import withStyles from '@mui/styles/withStyles';
+import CheckOutlinedIcon from '@mui/icons-material/CheckOutlined';
+import ClearOutlinedIcon from '@mui/icons-material/ClearOutlined';
+import DeleteForeverOutlinedIcon from '@mui/icons-material/DeleteForeverOutlined';
+import EditOutlinedIcon from '@mui/icons-material/EditOutlined';
+import InfoOutlinedIcon from '@mui/icons-material/InfoOutlined';
+import MessageOutlinedIcon from '@mui/icons-material/MessageOutlined';
+import { DateTimePicker } from '@mui/x-date-pickers';
```

Comment:
```suggestion
import DateTimePicker from '@mui/x-date-pickers/DateTimePicker';
```
If there's only one import, should we specify it directly with the path?

Reply:
No, see my above comment.

Reply:
Well, technically a different question than the one above since you wrote two different ways of doing the same thing. I'm just trying to figure out what is the best practice for the long term and then get consistent with it. Whatever you do, could you please write it the same way for both the TimePicker and DateTimePicker imports in this PR?

Reply:
I'm not sure what you mean about different ways - they're both named imports, aren't they? If you're referring to the different import path, I don't think that really matters.

Honestly, these imports are auto-generated for me while I'm writing code so I didn't actually write either of them by hand.